kalite/updates/management/commands/languagepackdownload.py

```diff
@@ -160,11 +159,17 @@ def move_dubbed_video_map(lang_code):
             logging.error("Error removing dubbed video directory (%s): %s" % (dubbed_video_dir, e))
 
 def move_video_sizes_file(lang_code):
-    """
-    This is no longer needed. See:
-    https://github.com/learningequality/ka-lite/issues/4538#issuecomment-144560505
-    """
-    return
+    lang_pack_location = os.path.join(settings.USER_WRITABLE_LOCALE_DIR, lang_code)
```

Comment:
This also appears to be a regression, but I'm not sure. (I wrote the patches for the above two cases, hence my certainty.)